client/my-sites/stats/checklist-banner/index.jsx

```diff
@@ -0,0 +1,136 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import Gauge from 'components/gauge';
+import ProgressBar from 'components/progress-bar';
+
+export class ChecklistBanner extends Component {
+	static propTypes = {
+		task: PropTypes.shape( {
+			id: PropTypes.string,
+			title: PropTypes.string,
+			description: PropTypes.string,
+			image: PropTypes.string,
+		} ),
+		completed: PropTypes.number,
+		total: PropTypes.number,
+		onClick: PropTypes.func,
+	};
+
+	static defaultProps = {
+		task: null,
+		onClick: noop,
+		completed: 0,
+		total: 1,
+	};
+
+	handleClick = () => {
+		this.props.onClick( this.task.id );
+	};
+
+	getTask() {
+		const { completed, total, translate } = this.props;
+
+		if ( completed === total ) {
+			return {
+				id: 'ready-to-share',
+				title: translate( 'Your site is ready to share' ),
+				description: translate(
+					'We did it! You have completed {{a}}all the tasks{{/a}} on our checklist.',
+					{
+						components: {
+							a: <a href="" />,
+						},
+					}
+				),
+				image: '/calypso/images/stats/tasks/ready-to-share.svg',
+			};
+		}
+
+		return this.props.task;
+	}
+
+	renderShareButtons() {
+		return <div className="checklist-banner__actions">Share buttons will be here!</div>;
+	}
+
+	renderTaskButton() {
+		return (
+			<div className="checklist-banner__actions">
+				<Button onClick={ this.handleClick } className="checklist-banner__button" primary>
+					{ this.props.translate( 'Do it' ) }
+				</Button>
+				<a href="" className="checklist-banner__link">
+					{ this.props.translate( 'View your checklist' ) }
+				</a>
+			</div>
+		);
+	}
+
+	render() {
+		const { completed, total, translate } = this.props;
+		const task = this.getTask();
+		const percentage = ( completed / total * 100 ).toFixed( 1 );
+
+		if ( ! task ) {
+			return null;
+		}
+
+		return (
+			<Card className="checklist-banner">
+				<Gauge
+					width={ 152 }
+					height={ 152 }
+					lineWidth={ 18 }
+					percentage={ +percentage }
+					metric={ translate( 'completed' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Show completed' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/checklist-banner-to-stats-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).